ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/ExampleClass.java

```diff
@@ -0,0 +1,6 @@
+package edu.usb.argos.ASTProcessor;
+
+public class ExampleClass {
```

Comment:
This class is only for testing purposes and is used to verify the functionality of `JavaParser`, maybe should be within the `tests` section.

Reply:
According to what was discussed in class with the trainer, the conclusion was to eliminate the tests since the pull requests will already comply with the tests that are needed and this will only have the initialization of the javaparser library.